### ISSUE TITLE
[Suggestion] Improvements to the Package (focusNode and onTap)

### ISSUE THREAD

**Issue by offkevyn** [AbdullahChauhan/custom-dropdown#82]
I would like to suggest two improvements for the dropdown package available in this repository. It would be very useful to add the following properties:

1. **focusNode**: Allows greater control over the focus of the dropdown, facilitating focus management in complex forms.
2. **onTap**: A property that enables actions to be executed when the dropdown is clicked, offering greater flexibility for user interactions.

I believe the inclusion of these properties would make the package even more robust and functional for various use cases.

Thank you for the excellent work and for considering my suggestion.

Best regards,
offkevyn

**Comment by mohmdramadn** [comment]
Is there any update on these suggestions ?

**Comment by Mirajul-Mohin** [comment]
Yes, it would be great if the dropdown's search field could be automatically focused when opened. It would allow the user to type text directly in search field.
